### PR TITLE
Over fitting

### PR DESCRIPTION
**Pull request by yoheitaonishi**

### CODE REVIEW

**Comment by yoheitaonishi** [comment]
dropout 追加してみたけど過学習なおらなかった
ただ、以前やった時は特に過学習なってなかったんだが、kerasのバージョンをあげるとなる

**Comment by yoheitaonishi** [comment]
dropout層追加すると加工した動画ベースでは改善されているような気がする

**Comment by yoheitaonishi** [comment]
その他対策としては `正則化` させてみる。
`lr`の確認もする。

**Comment by yoheitaonishi** [comment]
[正則化](https://keras.io/ja/regularizers/)

[kernel regularizers vs activity regularizers](https://stackoverflow.com/questions/44495698/keras-difference-between-kernel-and-activity-regularizers)

**Comment by yoheitaonishi** [comment]
maxpooling層を正則化できないのはなぜだったか？

**Comment by yoheitaonishi** [comment]
https://github.com/naoya0731/deepmosaic/pull/1/commits/cedc4e2e2886259e107cb6f945522dbf31ca5884 本コミットは完全に改悪だった。

一回元の動画で同じようにできるかやってみる。
keras 2系にしたため。

もう一度確認したが、Dropout層は有効に利いているような感じ。

**Comment by yoheitaonishi** [comment]
l1はdropout層追加時とほとんど変化はみられない

**Comment by yoheitaonishi** [comment]
元動画でやってもうまくいかない

**Comment by yoheitaonishi** [comment]
SSD300にはDropout層がないっぽいので実はdropout追加していけないんじゃないか？という気になってきた

https://qiita.com/de0ta/items/1ae60878c0e177fc7a3a#3-%E5%AE%9F%E9%A8%93%E7%B5%90%E6%9E%9C

**Comment by yoheitaonishi** [comment]
`loc`と`conf`の違いは何か？
